ntia_conformance_checker/sbom_checker.py

```diff
@@ -64,7 +63,10 @@ def check_doc_author(self):
 
     def check_doc_timestamp(self):
         """Check for document timestamp."""
-        if self.doc.creation_info.created is None:
+        if (
+            self.doc.creation_info.created is None
+            or self.doc.creation_info.created == ""
+        ):
```

Comment:
I think the preferred Python way to check for empty strings would be `if not self.doc.creation_info.created:`

See https://stackoverflow.com/a/9573259

Reply:
Thank you, @goneall. Will fix.